doc/mdc-migration.md

```diff
@@ -0,0 +1,56 @@
+# Material Design Components (MDC) Migration
+## Status:
+Accepted on 2020-04-16
+
+## Context:
+We have been using `Theme.Design.*` theme from the start, while that worked perfectly it did not allow us to use new material components because those are dependent on `Theme.MaterialComponents.*` app theme. Using new material components allows us to have much more control over component theming like text styles, icon & icon gravity, strokes, corners, etc.,
+
+- Right now we are using 3 different kind of buttons to satisfy various design requirements `Button`, `OmegaCenterIconButton` & `PrimarySolidButton`. Moving to material components allow us to use single `MaterialButton`. It would also allow us to use different material components. You can find more components supported by Android and how to theme them [here](https://material.io/components).
```

Comment:
Just one minor change here. Can you move this line from here to `Decision`?
"Moving to material components allow us to use single `MaterialButton`. It would also allow us to use different material components. You can find more components supported by Android and how to theme them [here](https://material.io/components)."